pop-api/examples/fungibles/utils.rs

```diff
@@ -0,0 +1,196 @@
+// A set of helper methods to test the contract calls.
+
+use drink::{
+	session::{bundle::ContractBundle, error::SessionError, Session, NO_SALT},
+	DispatchError,
+};
+use pop_api::primitives::{AccountId, TokenId};
+use pop_sandbox::{AccountId32, Balance, Sandbox, INIT_VALUE};
+use scale::{Decode, Encode};
+
+use super::*;
+
+// PSP22 functions.
```

Comment:
If we only use them once, these constants are unnecessary

Reply:
We use it in the `expect_call_reverted` as well, let see if I can refactor the method.